.nais/naiserator-prod.yaml

```diff
@@ -51,6 +51,7 @@ spec:
   gcp:
     sqlInstances:
       - type: POSTGRES_15
+        tier: db-custom-1-3840
```

Comment:
Er det denne som fører til nedetid? 😊

Reply:
Ja, gjorde det på ismeroppfolging før i dag: Den haltet litt i et minutt eller to. Men tror kanskje dette tar lengre tid på de app'ene som allerede har en ganske stor database.